contributions/essay/mrhasa-ved.megha/README.md

```diff
@@ -0,0 +1,25 @@
+# Essay: Comparison of Azure, AWS, and GCP based on DevOps.
+
+### Members
+
+Megha Gangwal (ved.megha@gmail.com)
+GitHub: [gangwalmegha](https://github.com/gangwalmegha/)
+
+Md Rezaul Hasan (mrhasa@kth.se)
+GitHub: [MDRezaulHasan](https://github.com/MDRezaulHasan/)
+
+### Proposal
+
+We are planning to do a comparison of Azure, AWS, and GCP based on DevOps. We will do a comparison of each setup on different parameters(to be decided).
+
+### Topics to covered
+
+1. Introductions to DevOps
+2. Different DevOps Solutions on Public Cloud, Private Cloud
+3. Reasons for selecting AWS, GCP, and Azure for comparison
+4. Description of DevOps set up on each of three clouds
+5. Compare with large scale storage
+6. CI/CD Pipeline pros and cons in different platform
+7. Comparison of each setup on the parameters like service integration
```

Comment:
It is not clear what you mean by "on the parameters like service integration". You want to compare which features of them? Their speed? Their popularity? The tools that can be easily integrated into them? The limits they have (regarding project size, number of builds, ...)?